src/hometask5/near_power_of_two.py

```diff
@@ -0,0 +1,15 @@
+# Написать программу которая находит ближайшую степень двойки
+# к введенному числу. 10(8), 20(16), 1(1)
+
+import math
+
+
+def near_power_of_two(num):
```

Comment:
it was task for binary shift ^)

Reply:
Made new solution, left and commented the old one